index.html

```diff
@@ -124,5 +124,30 @@
     <p class="navbar-cta navbar-item right" id="navbar-cta-text">We're hiring</p>
   </nav>
 
+  <div id="hiring-and-aearly-access-section">
+    <div id="join-hiring" class="card-subsection">
+      <h3> We're Hiring </h3>
+      <p> Help us redefine the calendar, reclaim our schedules, and make the time we spend together, better </p>
+      <a href="#" class="btn roles-button">See the roles</a>
+    </div>
+
+    <div id="picture-join">
+      <img id="flying-shoe" class="above-bg shake" src="images/joinbox-flying-shoe.png" alt="flying shoe">
```

Comment:
Don't use abbreviations for class names. These always leave unclear what it means. What is a bg? If someone has to ask what it means, then it's not clear enough.